pydis_site/apps/api/models/bot/metricity.py

```diff
@@ -43,3 +45,28 @@ def total_messages(self, user_id: str) -> int:
             raise NotFound()
 
         return values[0]
+
+    def total_message_blocks(self, user_id: str) -> int:
+        """Query number of 10 minute blocks the user has been active during."""
```

Comment:
Don't end a sentence with a preposition - especially not the preposition "during", that's just weird, dude.

Reply:
Also, maybe you can describe some of the context for this? We should document some context _somewhere_, and I think this is the best option we have.